xbean-finder/src/test/java/org/apache/xbean/finder/archive/JarArchiveTest.java

```diff
@@ -63,12 +64,12 @@ public void setUp() throws Exception {
     public void testGetBytecode() throws Exception {
 
         for (Class clazz : classes) {
-            assertNotNull(clazz.getName(), archive.getBytecode(clazz.getName()));
+            Assert.assertNotNull(clazz.getName(), archive.getBytecode(clazz.getName()));
```

Comment:
Shouldnt change, please fix the unexpected diff parts

Reply:
this would be single line without 'Assert.' prefix in whole test, let us leave it as is

Reply:
If I'm not mistaken it is the exact opposite, you added Assert everywhere (maybe your IDE tricked you?) instead of following the pattern in place in the class so I'm just asking you to not change the code you don't modify and instead of forcing the existing code to comply to your settings, to align your settings/coding style on the existing code.

Reply:
there is a setting in my ide "Alter ONLY manually edited code", and seems it doesn't work as i expect.
btw is there an Eclipse config file with code style settings for this project?

Reply:
Sadly I think geronimo misses some shared config (to be frank I think it is/was also a good compromise but understand it can miss for contributions)